content/en/docs/concepts/services-networking/service.md

```diff
@@ -203,6 +204,10 @@ subsets:
 The name of the Endpoints object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
+When you create an [Endpoints](docs/reference/kubernetes-api/service-resources/endpoints-v1/)
+object for a Service, you set the name of the new Endpoints object to be the same as the
+name of the Service.
```

Comment:
```suggestion
of the Service.
```

Reply:
“the same as the of the Service” doesn't look idiomatic to me @divya-mohan0209 - is that definitely the change you wanted to suggest?

Reply:
I meant to suggest "the same as that of the Service".